src/common.js

```diff
@@ -56,6 +56,28 @@ export const languages = [
       tabSize: 2,
     },
   },
+  {
+    id: "typescript",
+    name: "TypeScript",
+    panel: "code",
+    extensions: [".ts", ".mts"],
```

Comment:
It is, in fact, a thing.

> In turn, TypeScript supports two new source file extensions: .mts and .cts. When TypeScript emits these to JavaScript files, it will emit them to .mjs and .cjs respectively.

source: https://devblogs.microsoft.com/typescript/announcing-typescript-4-5-beta/